article/sections/_3_graph_overview.tex

```diff
@@ -49,20 +49,17 @@ \section{Overview of Causal Graphs}
 Figure \ref{fig:simple-graph-confounded} shows an indirect connection between Z and Y that goes through U.
 Therefore, even if we condition on X, omitting U will yield biased results of the relationship between Z and Y, due to the variation in both Z and Y caused by U.
 The structural equations also show that the ignorability assumption of Z does not hold if we only control for X, and we risk obtaining biased estimates of the causal effect of Z on Y if we fail to account for U, sometimes referred to as a common cause.
-Even in this very simple example with only three or four variables, we gain an advantage in parsimony and expressiveness by using a causal graph to encode assumptions.
-The conciseness of DAGs becomes even more useful in real-world applications.
-Here, applied problems may have many available covariates and a complex causal structure.
-In such problems, fully writing out the structural equations for the system may be tedious, and readers may still find it hard to understand.
+While the set of structural equations and the DAGs illustrate the same points in theory, one may rightfully argue that as the complexity of the problem increases, with many covariates available for the modeler to choose from, a graphical representation of the relationship between those covariates quickly becomes more intuitive and easier to follow. 
```

Comment:
I think the sentence should be split so that it doesn't express many thoughts at once. Here's an example:
```suggestion
While the set of structural equations and the DAGs illustrate the same points in theory, the clarity of structural equations scales poorly with a problem's complexity. 
As the number of covariates in one's causal system increases, a graphical representation of the relationship between those covariates quickly becomes more intuitive and easier to follow.
```